docs/Examples.md

```diff
@@ -729,3 +738,78 @@ spec:
         secretName: splunk-cluster-master
   defaultsUrl: /mnt/cluster-master/default.yml
 ```
+## Managing global kubernetes secret object
+
+### Creating global kubernetes secret object
+
+To create the global kubernetes secret object you can use the kubectl command line tool. Here are the steps to the same:
+
+1. Retrieve the namespace in the current context using the following command `kubectl config view --minify --output 'jsonpath={..namespace}'` and make a note of it.
+If the command doesn't display an output it indicates that we are in the `default` namespace.
```

Comment:
If the intention of these two lines is to get into the namespace, where we need to create a secret object, then we can just highlight the namespace switch command?

Reply:
The intention of these two lines to retrieve the current namespace in the context. If none is configured we use default.

Reply:
I may be wrong in my understanding, but my point is, it is not much clear, if somebody wants to create a secret object for a given namespace of choice.  For example, say, the customer has 4 namespaces, namely n1, n2, n3 and n4. If he wants to change the keys for the deployments in namespace n3, he has to switch the namespace context using the command "kubectl config set-context --current --namespace=n3", (OR i am not sure if there is a better alternative) then he can create/modify a secret object.